src/actions/content.js

```diff
@@ -100,6 +100,178 @@ function* loadContent(action) {
   }
 }
 
+export const CONTENT_SAVE = 'CONTENT_SAVE';
+export const contentSave = content => {
+  return {
+    type: CONTENT_SAVE,
+    payload: {
+      content,
+    },
+  };
+};
+
+export const CONTENT_DELETE = 'CONTENT_DELETE';
+export const contentDelete = content => {
+  return {
+    type: CONTENT_DELETE,
+    payload: {
+      content,
+    },
+  };
+};
+
+export const ACTION_EXECUTE = 'ACTION_EXECUTE';
+export const actionExecute = (action, nids) => {
+  return {
+    type: ACTION_EXECUTE,
+    payload: {
+      action,
+      nids,
+    },
+  };
+};
+
+export const SUPPORTED_ACTIONS = [
+  'entity:delete_action:node',
+  'node_make_sticky_action',
+  'node_make_unsticky_action',
+  'node_promote_action',
+  'entity:publish_action:node',
+  'node_unpromote_action',
+  'entity:unpublish_action:node',
+];
+
+// @todo Find a better name
+// @todo How do we update the store with the new values of the nodes
+//    or the deleted nodes.
+export function* doActionExecute({ payload: { action, nids } }) {
```

Comment:
Sorry I'm not sure what you mean with that. Do you suggest we should have some form of VBO plugin system in JS?

Reply:
I talked with Wim and Gabe and they mentioned that JSON API will support bulk operations in future. I assume it means instead of sending separate HTTP requests for each action, we would be sending multiple requests in a single HTTP request.

Reply:
Oh that makes sense! Sure, let's add a quick todo